drush.services.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  lightning.commands:
+    class: \Drupal\lightning\Commands\LightningCommands
```

Comment:
Nit: For consistency, can this be `class: 'Drupal\lightning\Commands\LightningCommands'`?